_data/tools.yml

```diff
@@ -1192,10 +1192,18 @@
 - name: kin-openapi
   category:
     - parsers
+  github: &kin-openapi_URL https://github.com/getkin/kin-openapi
```

Comment:
This is cool and all but you can just add multiple categories if you like? Not sure if the whole toolchain is happy with such fancy YAML. :D

Reply:
Hi, sorry for taking this long to answer. I just saw this.

I'm not just adding multiple categories because, if you look closely, you'll see in the first it concerns both v2 and v3 whereas it has data-validators only for v3.

Anchors are 18 years old now https://yaml.org/spec/1.0/index.html#id2563853 yes very fancy YAML indeed